### PR TITLE
update descriptions of expressions supported to be pushed down

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
- update descriptions of expressions supported to be pushed down in "functions-and-operators/expressions-pushed-down.md".
- revise the order of `tidb_sysdate_is_now` in "system-variables.md".

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/8684

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1552,6 +1552,12 @@ set tidb_slow_log_threshold = 200;
 - 范围：`[0, 9223372036854775807]`
 - 这个变量用于限制 TiDB 同时向 TiKV 发送的请求的最大数量，0 表示没有限制。
 
+### `tidb_sysdate_is_now`（从 v6.0.0 版本开始引入）
```
Comment (TomShawn) [inline system-variables.md:1555]: 系统变量文档里有两个 `tidb_sysdate_is_now`（从 v6.0.0 版本开始引入），请删除重复的

**Comment by TomShawn** [comment]
@en-jin19 请@ tech review 并尽早推进合并

**Review by Yisaer — APPROVED** [review]

**Pull request merged**